### PR TITLE
Update bookdown

### PR DESCRIPTION
**Pull request by ivelasq**
Thanks!

### CODE REVIEW

File: 01-retitle.Rmd [hunk 1]
```diff
@@ -929,7 +929,7 @@ This chapter explores what aggregate data is, and how to access, clean, and expl
 
 ### Cohort 1
 
-TBD
+`r knitr::include_url("https://www.youtube.com/watch?v=6k_F6XGEmqs")`
```
Comment (tanho63) [inline 01-retitle.Rmd]: Don't forget that you need the embed link for these to work correctly!

File: 01-retitle.Rmd [hunk 2]
```diff
@@ -950,4 +950,42 @@ TBD
...
+
+## Slides
+
+`r knitr::include_url("./R/2021-04-07/walkthrough-5.html")`
+
+## Meeting Videos
+
+### Cohort 1
+
+`r knitr::include_url("https://www.youtube.com/watch?v=6k_F6XGEmqs")`
```
Comment (tanho63) [inline 01-retitle.Rmd]: also embed

**Review by tanho63 — CHANGES_REQUESTED** [review]
I `think` we talked about the embed link thing from last time but if not, let me know and I can help you find them!

**Comment by ivelasq** [comment]
Ah, thanks! I think I fixed them all...

**Comment by tanho63** [comment]
@ivelasq can you also hit the very first one at line 19 while you're on this? :)

**Comment by ivelasq** [comment]
Nice catch! Thanks.

**Comment by tanho63** [comment]
Working through some GHA issues (unrelated to this PR)

-
Tan Ho
________________________________

**Review by tanho63 — APPROVED** [review]

**Pull request merged**